### PR TITLE
Add variable trigger to delivery job

### PR DESCRIPTION
**Pull request by EleanorRigby**
[ci skip]

[Delivers #152057499](https://www.pivotaltracker.com/story/show/152057499)

### CODE REVIEW

File: ci/Rakefile [hunk 1]
```diff
@@ -74,6 +74,10 @@ task :pipeline, [:name] => 'pipeline.yml' do |t, args|
   # Get the git branch name
   branch = ENV.fetch('BRANCH') { `git symbolic-ref -q --short HEAD`.strip }
 
+  # Set the trigger for delivery job based on which branch it is.
+  # Only master branch should trigger delivery jobs.
+  trigger = (branch == 'master' ? 'true': 'false')
```
Comment (ktchen14) [inline ci/Rakefile]: Is this variable used?

Comment (EleanorRigby) [reply ci/Rakefile]: Removed this.

File: ci/Rakefile [hunk 2]
```diff
@@ -87,6 +87,8 @@ task :pipeline, [:name] => 'pipeline.yml' do |t, args|
 
   # Default to master if the branch name is indeterminate (because HEAD is
   # detached).
+  # Set the trigger for delivery job based on which branch it is.
```
Comment (ktchen14) [inline ci/Rakefile]: This comment should probably be moved to immediately before line 101.

Comment (EleanorRigby) [reply ci/Rakefile]: Moved

**Pull request merged**